.github/workflows/codeql-analysis.yml

```diff
@@ -64,9 +63,12 @@ jobs:
     #    and modify them (or add more) to build your code if your project
     #    uses a compiled language
 
+    - name: Update repository metadata
+      run: |
+        sudo apt-get update
     - name: Install dependencies
       run: |
-        sudo apt-get update && sudo apt-get --no-install-recommends install -qq -y \
```

Comment:
why is the update gone?

Reply:
It wasn't removed, just moved to a separate step above this one, which is not visible in this context for some reason.

Reply:
ah, totally missed that, sorry!